File: BACKEND/Application/GameSessions/Responses/CreateGameSessionResponse.cs
using Common.Enums.GameSession;
using Domain.GameSession;

namespace Application.GameSessions.Responses
{
    public class CreateGameSessionResponse
    {
        public Guid SessionId { get; set; }
        public required string SessionCode { get; set; }
        public required GameSessionSettings Settings { get; set; }
        public DateTimeOffset CreatedAt { get; set; }
        public GamePhase CurrentPhase { get; set; }
        public Guid HostUserId { get; set; }
        public int PlayersCount { get; set; }
    }
}

File: FRONTEND/backgammon-frontend/src/app/features/game-session/models/api/responses/create-game-session.response.ts
import { GameSessionSettings } from '../../game-session-settings.model';

export interface CreateGameSessionResponse {
  sessionId: string;
  sessionCode: string;
  settings: GameSessionSettings;
  createdAt: string;
  currentPhase: number;
  hostUserId: string;
  playersCount: number;
}
